src/MOI_wrapper.jl

```diff
@@ -420,6 +423,64 @@ function MOI.get(::Optimizer, ::MOI.ListOfOptimizerAttributesSet)
     return attributes
 end
 
+function MOI.compute_conflict!(o::Optimizer)
+    if o.conflict_status != MOI.COMPUTE_CONFLICT_NOT_CALLED
+        error("Conflict computation is destructive for the model and cannot be called twice.")
+    end
+    # free the transformed problem first
+    if LibSCIP.SCIPgetStage(o) != LibSCIP.SCIP_STAGE_PROBLEM
+        @SCIP_CALL LibSCIP.SCIPfreeTransform(o)
+    end
+    # we need names for all constraints
+    for (F, S) in MOI.get(o, MOI.ListOfConstraintTypesPresent())
+        for (idx, c_index) in enumerate(MOI.get(o, MOI.ListOfConstraintIndices{F,S}()))
+            if MOI.get(o, MOI.ConstraintName(), c_index) == ""
```

Comment:
This won't work for `VariableIndex` constraints

Reply:
these are currently excluded, because SCIP treats these as variable information and not as constraints, will need further adaptation

Reply:
I adapted the code to convert all variable constraints to scalar affine functions to be able to compute their infeasibility too